shared/src/main/scala/effekt/symbols/symbols.scala

```diff
@@ -141,14 +141,37 @@ package object symbols {
    *
    * Refined by typer.
    */
-  case class CallTarget(name: Name, symbols: List[Set[BlockSymbol]]) extends BlockSymbol
+  case class CallTarget(name: Name, symbols: List[Set[BlockSymbol]]) extends Synthetic with BlockSymbol
```

Comment:
Can you explain why `CallTarget` became a `Synthetic` symbol? Does this change how `CallTarget` is/(should be) used in other parts of the code?

Reply:
No, it doesn't change anything. It actually also doesn't have any meaning, besides being a marker (at the moment). Namer introduces `CallTarget`  symbols that don't have a concrete counterpart in the source language. They are only used to represent ambiguous calls, before typer goes over it and replaces it by the correct (resolved) function / blocksymbol.